### ISSUE TITLE
CMS [ZOOM]: Content SHOULD not be cut off when zoomed to 400% 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6399]
#  [508-defect-2 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Content **SHOULD** not be cut off when zoomed to 400%, which tests for user magnification tools and mobile design. See screenshot below. 

The div shown below is broken off when screen is zoomed to 400% and viewport is at 1280px (per the accessibility test process)

```html
<div class="vads-u-flex--auto">
    You have the option to add new evidence that a judge will review.
  </div>
```

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- As a screen magnifier user, I want to read all the content of the page without it being cut off.

- As a mobile user, I want to read all the content of the page without it being cut off.

## Environment

* Operating System: any 
* Browser: any
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/decision-reviews/board-appeal/` in browser
2. Set viewport width to 1280px wide
3. Zoom to 400%
4. Verify that `You have the option to add new evidence that a judge will review.` is cut off on the right side.

## Possible Fixes (optional)

When I removed the class `vads-u-flex--auto` the content reflowed within the viewport.

## Screenshots or Trace Logs

![vsa-av-zoom400-issue](https://user-images.githubusercontent.com/57469/75804183-97de1380-5d4d-11ea-9f28-f6adeecb79ec.png)

**Comment by RLHecht** [comment]
@ncksllvn I think this is more a you and @kelsonic thing than me...

**Comment by ncksllvn** [comment]
Fix will go live today

**Comment by jenstrickland** [comment]
**Rad, thank you!** Ping me in Slack when these two tickets are live, so I can validate, comment, and close the tickets.

**Comment by jenstrickland** [comment]
Fix validated. Thank you!
